### PR TITLE
VIH-7007 send hearing amendment email on hearing update

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-7007 automate hearing amendment email for Individuals
VIH-7157 automate hearing amendment email for Reps
VIH-7199 automate hearing amendment email for JOHs

### Change description ###

send hearing amendment email on hearing update to:

* JOHs
* LIPs
* Representatives

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs [hunk 1]
```diff
@@ -184,15 +184,15 @@ public async Task<ActionResult<HearingDetailsResponse>> EditHearing(Guid hearing
...
             try
             {
-                hearing = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
+                originalHearing = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
             }
             catch (BookingsApiException e)
             {
                 _logger.LogError(e,
-                    "Failed to get hearing {hearing}. Status Code {StatusCode} - Message {Message}",
+                    "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
```
Comment (williambowshersolirius) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:195]: shouldn't this be originalHearing?

Comment (williambowshersolirius) [inline AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:195]: Where is Hearing declared? as it would appear this should be originalHearing..

**Review by williambowshersolirius — COMMENTED** [review]
Just one comment on BookNewHearing action in HearingsController

File: AdminWebsite/AdminWebsite/Controllers/HearingsController.cs (hunk 1 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:195]: This is the property name we want to give when we look in app insights

Comment (williambowshersolirius) [reply AdminWebsite/AdminWebsite/Controllers/HearingsController.cs:195]: all good

**Review by shaed-parkar — COMMENTED** [review]
confirm is ready to use. just needs to be called on booking and adding new participants

File: AdminWebsite/AdminWebsite.AcceptanceTests/Features/BookingDetails.feature [hunk 2]
```diff
@@ -11,8 +11,7 @@ Scenario: Bookings Details for single day hearing
 	Then the hearing is available in video web
 	And the conference details match the hearing
 	And the user views the booking conference phone details
-
-@Smoketest @Smoketest-Extended
+
```
Comment (npoxon) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Features/BookingDetails.feature:14]: @shaed-parkar can you add this back in?

**Review by npoxon — CHANGES_REQUESTED** [review]

File: AdminWebsite/AdminWebsite.AcceptanceTests/Features/BookingDetails.feature (hunk 2 above)
Comment (shaed-parkar) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Features/BookingDetails.feature]: done

**Review by npoxon — APPROVED** [review]

**Pull request merged**